Simplified/NYPLSettingsAccountDetailViewController.m

```diff
@@ -213,26 +213,37 @@ - (void)viewDidLoad
   self.view.backgroundColor = [NYPLConfiguration backgroundColor];
   self.tableView.keyboardDismissMode = UIScrollViewKeyboardDismissModeInteractive;
 
+  UIActivityIndicatorView *activityIndicator = [[UIActivityIndicatorView alloc] initWithActivityIndicatorStyle: UIActivityIndicatorViewStyleGray];
```

Comment:
by moving the set up and start of the activity indicator here there's a possibility for it to start spinning even in the case at line 233 i.e. when we just have to set up the views. If there is a condition where we just have to call `setupViews` and we're done, we could simply `return` early and then do the activity indicator setup.